### DISCUSSION TITLE
네이버 잔여백신 도중 오류

### DISCUSSION THREAD

**Discussion by dadaid** [changdoc/naver-vaccine-macro#179] [Q&A]
![image](https://user-images.githubusercontent.com/88299833/127797870-db0f641c-792a-435f-a3c0-b44b286697ae.png)

밑에 글 올리신 분과 같은 오류가 발생해서 몇 번 지웠다 까는데도 똑같은 오류가 발생하네요

network 탭의 graphql status가 200이 안 뜨니 그냥 url 추출해서 예약 하는 것도 안됩니다 ㅠ 이런 경우는 어떻게 해야 할까요?
CORS missingalloworiginheader 라고 뜨는데 뭐가 문제일까요..

**Comment by changdoc** [comment]
에러난 요청 Header 값 내용을 보여주시면 좋을것 같은데요.
아래는 정상일때 인데요.

Request Headers 하위에 origin, referer의 최상위 도메인이 m.place.naver.com 으로 일치합니다.
CORS는 저부분이 일치하지 않을때 발생하는것으로 저는 알고 있는데 제 pc에선 재현이 안되네요.
![image](https://user-images.githubusercontent.com/1879494/127800031-4a222f9f-ef7d-4b5d-9de9-99f18ce85bb1.png)
